isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/anwendungskern.adoc

```diff
@@ -0,0 +1,201 @@
+= Anwendungskern
+
+include::documentation-guide::partial$licence.adoc[]
+
+// tag::anwendungskern-definition[]
+
+Der xref:glossary::terms-definitions.adoc#glossar-anwendungskern[Anwendungskern] ist der zentrale Bestandteil eines Backends.
+Hier werden die fachlichen Funktionen der zugehörigen xref:glossary::terms-definitions.adoc#glossar-anwendung[Anwendung] realisiert.
+Der Aufbau des Anwendungskerns sowie seine Kommunikation mit anderen Schichten ist in <<systemarchitektur-anwendungskern,folgender Grafik>> dargestellt.
+
+[[systemarchitektur-anwendungskern]]
+.Einbettung des Anwendungskerns in die Systemarchitektur
+image::referenzarchitektur:software-technisch/backend/anwendungskern.dn.svg[width=80%,align=center]
+
+// end::anwendungskern-definition[]
+
+[[fachkomponenten]]
+== Fachkomponenten
+
+// tag::fachkomponente-definition[]
+
+*Fachkomponenten* setzen die Anwendungskomponenten aus der fachlichen Architektur um.
+Sie bilden damit den Schnitt der Anwendungskomponenten in die technische Architektur ab.
+Fachkomponenten beinhalten möglichst wenig Technik und trennen so Anwendungslogik von Technologie.
+
+TIP: Fachkomponenten sind der Schlüssel für gute Wartbarkeit und einfache Weiterentwickelbarkeit des Anwendungskerns.
+Die Trennung von Anwendungslogik und Technik ist deshalb auch in anderen Schichten der technischen Architektur vorgesehen, z. B. in der Persistenz.
+
+// end::fachkomponente-definition[]
+
+Von außen betrachtet besteht jede Fachkomponente aus einer Schnittstelle (Interface) und den von ihr verwendeten Geschäftsobjekten (Model) sowie Ausnahmen (Exceptions).
+Alle Zugriffe auf die Fachkomponente von außen, auch von anderen Fachkomponenten aus, geschehen über die Schnittstelle.
+
+Die *Schnittstelle* stellt eine Fassade (Facade) für mindestens eine Anwendungsfallklasse dar.
+Die Aufgaben der Schnittstelle sind:
+
+* Aufrufe von außen entgegenzunehmen,
+* Aufrufe an die entsprechenden Anwendungsfallklassen weiterzuleiten,
+* Antworten der Anwendungsfallklassen an den Aufrufer zurück zu übermitteln,
+* Ausnahmen der Anwendungsfallklassen in Ausnahmen der Schnittstelle zu übersetzen.
+
+*Geschäftsobjekte* dienen zur Kommunikation der Fachkomponente nach außen sowie zur Umsetzung der Fachlichkeit (z.B. für interne Berechnungen).
+Sie werden im Anwendungskern definiert, sind nicht persistent und können in der Nutzungsschicht wiederverwendet werden, sofern hierfür keine Anpassungen durch Mappings o.ä. nötig sind.
```

Comment:
wenn Geschäftsobjekte nicht persistent sind, so stellt sich natürlich die Frage wie man Geschäftsobjekte ggf. persistieren kann, (d.h. wie und an welcher Stelle man Geschäftsobjekte und Entitäten ineinander umwandeln würde). 
Ich weiß nicht, ob man das hier erklären muss, denn es führt ggf. zu weit.

Reply:
Für dieses Ticket out of scope. Ich nehme das Thema mit.